docs/user/security-log.rst

```diff
@@ -41,7 +46,15 @@ Authentication failures and successes are both tracked.
 
 To access your organization logs:
 
-- Click on :guilabel:`Organizations` from your user dropdown
-- Click on your organization
+- Click on :guilabel:`<Username dropdown>`
+- Click on :guilabel:`Organizations`
+- Click on :guilabel:`<Organization name>`
 - Click on :guilabel:`Settings`
 - Click on :guilabel:`Security Log`
+
+Your organization security log is a great place to check periodically to ensure there hasn't been unauthorized access to your organization.
+
+If you have any additional information that you wished the security log was capturing,
+you can always reach out to :doc:`/support`.
+
+.. seealso:: We have other security related information in our :doc:`/security` page.
```

Comment:
```suggestion
.. seealso:: 

    :doc:`/security`
      Security information related to our own platform, personal data treatment, how to report a security issue and more.
```

I think we have been using a style like this where we put the link in a definition list and write a small sentence about what it contains.

If you accept it, make sure it renders properly 😄

Reply:
Yes! This is a great pattern, it makes it easy to list many different related articles instead of having a lot of "seealso" boxes.